usagov/usagov-2021
# Usagov 364 federal agency index changes

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-364

## Description

On federal directory A-Z site page intro is no longer hard coded and content team can edit it as they wish. 
Content team also now has access to header and footer html fields for A-Z page. 

Testing:
** Change in config file - config/sync/views.view.federal_agencies.yml
** Change in twig file - web/themes/custom/usagov/templates/views-view-summary--federal-agencies.html.twig


On /agency-index:
1. Go to edit
2. Check to make sure header & footer html fields are there
3. Create/ modify the page intro field
4. Save
5. Check that page intro is now displayed above search


## Checklist for the Developer



## Checklist for the Peer Reviewers
